NEWS.rst

```diff
@@ -1,6 +1,16 @@
 History
 =======
 
+0.12 (2019-02-12)
+-----------------
+* Make L1 product by applying calibration corrections aka applycal (#194 - #198)
```

Comment:
This makes it sounds like it happens automatically, which might lead people astray.

Reply:
Fixed.